src/components/homePage/About.vue

```diff
@@ -31,68 +20,33 @@
 	margin: auto;
 	color: black;
 }
-@media screen and (max-width:1250px) {
+.about_section{
+	display: flex;
+	flex-direction: row;
```

Comment:
Flex is great, but it can result in overflows sometimes, like below:
![image](https://user-images.githubusercontent.com/41529410/119239121-0525c200-bb15-11eb-9d39-a65e2857b378.png)

I would recommend adding the following line so it dynamically switches to the mobile view when the viewport gets too narrow.
`flex-wrap: wrap-reverse;`
This way, we can remove lines 39-41 in the media query (and perhaps even more!)